src/downshift.js

```diff
@@ -952,6 +954,11 @@ class Downshift extends Component {
   }
 }
 
+// Polyfill your component so the new lifecycles will work with older versions of React:
+if (Component.prototype && Component.prototype.isReactComponent) {
```

Comment:
Add `/* istanbul ignore else (preact) */` above this line and that'll take care of coverage :+1:

Reply:
Done. Code coverage is 100% now.